core/arch/arm/include/optee_ffa.h

```diff
@@ -1,11 +1,11 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /*
- * Copyright (c) 2019, Linaro Limited
+ * Copyright (c) 2019-2021, Linaro Limited
  */
 
 /*
  * This file is exported by OP-TEE and is kept in sync between secure world
- * and normal world drivers. We're using ARM FF-A 1.0 EAC specification.
+ * and normal world drivers. We're using ARM FF-A 1.0 specification.
```

Comment:
There's already a 1.1-Alpha0 at https://developer.arm.com/documentation/den0077/latest, does this PR comply with it?

Reply:
Yes it does, in the sense that it's not incompatible with it.

Reply:
OK fine.